src/services/nginx/default.conf

```diff
@@ -6,8 +6,16 @@ upstream broker {
     server broker:8000;
 }
 
+map $sent_http_content_type $expires {
+    default                    off;
+    text/html                  epoch;
+    text/css                   max;
+    application/javascript     max;
+}
```

Comment:
These settings ensure that the main `html` page has `no-cache` set which forces the browser to check the `etag` and `Last-Modified` fields. 

For `css` and `javascript` we set the expiry to some distant time in the future as they are immutable so the browser can keep them for ever.